app/v1/models/response/Business.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import utils.NestedJsonReads
+import v1.models.domain.{IncomeSourceType, TypeOfBusiness}
+
+case class Business(typeOfBusiness: TypeOfBusiness, businessId: String, tradingName: Option[String])
+
+object Business extends NestedJsonReads {
```

Comment:
I've had a thought about how to do this all without nested JSON reads - let me know when you've done everything else and we can go through that together